### DISCUSSION TITLE
Multiple controllers

### DISCUSSION THREAD

**Discussion by interw-12** [mariusz-ostoja-swierczynski/tech-controllers#44] [Ideas]
Any chances for multiple controllers? I have two L4-Wifi controllers and I can see entities only one of them in my HA.

**Comment by mariusz-ostoja-swierczynski** [comment]
Yes, multiple controllers feature is already implemented in Dev branch. Please download, test and feedback if it works or not for you. Once I receive few passed tests I will merge it do master branch.
Cheers, Mariusz

**Comment by interw-12** [comment]
OK, I installed Dev branch. Now in integration I can see both of my L4-Wifi controllers, but I still don't have entities of the second one (I can see only entities of first). In comparation with Master branch now I have two additional enities [binary_sensor] (pump and wireless relay)

**Comment by chudystg** [comment]
I have the same problem with 5 devices. Only the first one is visible (in Dev branch)

**Comment by interw-12** [comment]
After I installed new (2.0) version everything works fine with my two L4-Wifi controllers and now I got a lot new entities like pump and wireless relay, battery, signal strenght etc.
